### PR TITLE
chore: Add a new NearOne telemetry endpoint alongside the old explorer telemetry

### PR DESCRIPTION
**Pull request by Trisfald**
Add a new telemetry endpoint hosted on N1 alongside the old telemetry endpoint hosted by Pagoda and based on the legacy explorer.  Part of #11156.

EDIT: adding the new endpoint without replacing the old one

### CODE REVIEW

**Comment by wacban** [comment]
@Trisfald I have no context on this, is there any other codeowner reviewer with more context? If not I'll be happy to review, but can you provide some more info about this change?

File: nearcore/res/example-config-gc.json [hunk 1]
```diff
@@ -24,7 +24,8 @@
     },
     "telemetry": {
         "endpoints": [
-            "https://explorer.mainnet.near.org/api/nodes"
+            "https://explorer.mainnet.near.org/api/nodes",
+            "https://telemetry.nearone.org/nodes/mainnet"
```
Comment (tayfunelmas) [inline nearcore/res/example-config-gc.json:28]: how are these multiple endpoints used by the code? the first one is primary and the other is fallback or does it send requests to both?

File: nearcore/res/example-config-gc.json [hunk 2]
```diff
@@ -24,7 +24,8 @@
     },
     "telemetry": {
```
Comment (wacban) [inline nearcore/res/example-config-gc.json:25]: I haven't seen this config before. Can you tell me what is it and how is it used? Is it identical to the default config generated by neard or to the config we upload to S3?

**Review by wacban — APPROVED** [review]
LGTM

File: nearcore/res/example-config-gc.json (hunk 1 above)
Comment (Trisfald) [reply nearcore/res/example-config-gc.json:28]: the requests are sent to both (see the [actor implementation](https://github.com/near/nearcore/blob/master/chain/telemetry/src/lib.rs#L80))

File: nearcore/res/example-config-gc.json (hunk 2 above)
Comment (Trisfald) [reply nearcore/res/example-config-gc.json:25]: I'm not sure myself how or when it is used. The config uploaded to S3 is another thing, I asked Andrei to change it manually.

**Pull request merged**